tests/Common/MetadataTest.php

```diff
@@ -1356,4 +1391,42 @@ public function testTypedTableMetadataRestrictions(): void
             $this->assertSame('storage.metadata.invalidProvider', $e->getStringCode());
         }
     }
+
+    public function provideComponentsClientTypeBasedOnSuite()
```

Comment:
Bylo by super to přesunout jinam, aby se to dali využívat i v jiných testcasech

Reply:
jj :( zatím jsem se zasekal v connectiona dál jsem do testů nesahal. Ono to tam nějak potřebuje $this a bylo to takové zamotané, tak jsem si říkal, že se to prostě zatím zkopíruje. V principu často stejně potřebuješ komplexnější kombinace v tom dataprovideru :(

Reply:
Vyřešeno